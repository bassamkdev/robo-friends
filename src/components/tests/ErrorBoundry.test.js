import React from 'react';

import ErrorBoundry from '../ErrorBoundry';

import { shallow } from 'enzyme';

it('render correctly ErrorBoundry component', () => {
    const wrapper = shallow(<ErrorBoundry children={'children'} />);
    expect(wrapper).toMatchSnapshot();
})

it('catches any error', () => {
    const mockError = new Error
    const wrapper = shallow(<ErrorBoundry children={'children'} />);
    wrapper.instance().componentDidCatch(mockError)
    expect(wrapper.state().hasError).toEqual(true)
})