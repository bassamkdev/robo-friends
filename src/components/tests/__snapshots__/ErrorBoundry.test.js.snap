// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly ErrorBoundry component 1`] = `"children"`;
